tests/utils/test_json.py

```diff
@@ -25,9 +25,26 @@
 import pendulum
 import pytest
 
+from airflow.datasets import Dataset
 from airflow.utils import json as utils_json
 
 
+class Z:
+    def __init__(self, x):
+        self.x = x
+
+    def serialize(self) -> dict:
+        return dict({"x": self.x})
+
+    @staticmethod
+    def deserialize(data: dict):
+        return Z(data["x"])
```

Comment:
I wonder if we should call these `airflow_serialize` etc, to reduce the risk of these clashing when used with "existing" objects?

Reply:
My idea is that serialization isn't something special and not necessarily tied to Airflow. In essence it is just a to_dict()/from_dict() which goes for anything in python.